kybra/__init__.py

```diff
@@ -184,6 +187,10 @@ def canister_balance() -> nat64:
     def canister_balance128() -> nat:
         return _kybra_ic._kybra_canister_balance128() # type: ignore
 
+    @staticmethod
+    def clear_timer(timer_id: TimerId) -> None:
```

Comment:
Let's match Azle/Kybra exactly, in Azle the parameter is `id`. We should choose one and use it in both projects.